runtime/lua/vim/keymap.lua

```diff
@@ -0,0 +1,136 @@
+local keymap = {}
+
+--- Add a new keymap.
+--- Examples:
+--- <pre>
+---   -- You can make keymaps worth Lua functions
```

Comment:
What does "worth" mean in this context? Maybe re-word this to simply
```suggestion
---   -- You can use Lua functions
```

Reply:
typo *with 🤦‍♂️